usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/drawSphereClient.js

```diff
@@ -829,10 +833,14 @@
 
         onScaleChanged: function(){
             var sphereProperties = Entities.getEntityProperties(_this.entityID, ['dimensions']);
-        
             currentStrokeWidth = sphereProperties.dimensions.x;
         },
 
+        onSkeletonModelURLChanged: function(){
+            Entities.callEntityMethod(_this.colorPaletteID, "createPaintSphere");
```

Comment:
does it matter that this call is before the deleteEntity() call below?

Reply:
It was supposed to be so to ensure it happens before this paint sphere gets deleted.

Reply:
OK - I _think_ that is safe.